sql/server.go

```diff
@@ -145,7 +145,7 @@ func (p parameters) Arg(i int) (parser.Datum, bool) {
 	case *string:
 		return parser.DString(*t), true
 	default:
-		panic(fmt.Sprintf("Incorrect type %T", t))
+		panic(fmt.Sprintf("Unexpected type %T", t))
```

Comment:
Generally error messages are uncapitalized. I extend this to panics as well. `s/Unexpected/unexpected/g`